backfill.py

```diff
@@ -62,17 +78,17 @@ def fetch_nightly(download_date):
         return None
 
     # TODO: Could add build type, architecture, etc...
-    return {KEY_NAME: filename, KEY_DATETIME: download_date}
+    return {KEY_NAME: filename, KEY_DATETIME: download_date, KEY_COMMIT: "", KEY_ARCHITECTURE: architecture}
 
 
 def get_date_array_for_range(startdate, enddate):
     delta_dates = (enddate - startdate).days + 1
     return [startdate + timedelta(days=i) for i in range(delta_dates)]
 
 
-def download_nightly_for_range(array_of_dates):
+def download_nightly_for_range(array_of_dates, artchitecture):
```

Comment:
thought (non-blocking): it's a bit disconcerting to run such a long script without any output to see how it's progressing. In a later PR, maybe we should log for each download/build and each perf test run. For example:
```
(1/2) Downloading Nightly 2021.07.30…
(2/2) Downloading Nightly 2021.07.31…

(1/2) Running perf test for Nightly 2021.07.30…
```